src/postGithubComments.ts

```diff
@@ -6,12 +6,12 @@ import pu = require("./utils/packageUtils");
 
 const { argv } = process;
 
-if (argv.length !== 10) {
-    console.error(`Usage: ${path.basename(argv[0])} ${path.basename(argv[1])} <user_to_tag> <pr_number> <comment_number> <distinct_id> <is_top_repos_run> <result_dir_path> <artifacts_uri> <post_result>`);
+if (argv.length !== 11) {
```

Comment:
Given that this can also run user-tests if `is_top_repos_run` is false, do you still need to permit 10?

Reply:
No, the arg is always specified by the pipeline file, it just doesn't do anything for the user tests.